src/glyph/mod.rs

```diff
@@ -251,6 +252,49 @@ impl Contour {
     fn is_closed(&self) -> bool {
         self.points.first().map_or(true, |v| v.typ != PointType::Move)
     }
+
+    /// Converts the contour to a kurbo::BezPath
+    #[cfg(feature = "kurbo")]
+    pub fn to_kurbo(&self) -> Result<kurbo::BezPath, ErrorKind> {
+        let mut path = kurbo::BezPath::new();
+        let mut offs: VecDeque<kurbo::Point> = VecDeque::new();
+        // Add end-of-contour offcurves to queue
+        let mut i = self.points.len() - 1;
+        while i > 0 && self.points[i].typ == PointType::OffCurve {
+            let kurbo_point = kurbo::Point::new(self.points[i].x as f64, self.points[i].y as f64);
+            offs.push_front(kurbo_point);
+            i -= 1;
+        }
+        if self.is_closed() {
+            path.move_to(kurbo::Point::new(self.points[i].x as f64, self.points[i].y as f64));
+        }
+        for pt in &self.points {
+            let kurbo_point = kurbo::Point::new(pt.x as f64, pt.y as f64);
```

Comment:
I'd probably go ahead and add `CountourPoint::to_kurbo` while we're in here?

Reply:
Ah! I tried to do something like this but it didn't quite work, so I gave up. The `path.<operator>_to` methods take anything which is `Into<kurbo::Point>`, so I thought it would be much neater to implement that trait for `ContourPoint`. But I guess that consumes the point, and I ended up in a fight with the borrow checker. `to_kurbo`, making a copy, makes sense.